### PR TITLE
CUP Range Rover nodes

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    Bob asked for them, Bob gets them.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Comment by jaj22** [comment]
Should this have canLoadWeapon=0, or is it open-topped?

**Comment by Bob-Murphy** [comment]
> Should this have canLoadWeapon=0, or is it open-topped?

currently loading in to check exactly that as I think that there are benches in the way

File: A3A/addons/Logistics/Nodes/CUP.hpp [hunk 1]
```diff
@@ -330,3 +330,25 @@ class CUP_WheeledVehicles_CUP_WheeledVehicles_Datsun_CUP_datsun_civil_p3d : TRIP
         };
     };
 };
+
+class CUP_WheeledVehicles_CUP_WheeledVehicles_LR_model_LeftHand_CUP_LR_p3d: TRIPLES(ADDON,Nodes,Base)
+{
+        class Nodes
```
Comment (Bob-Murphy) [inline A3A/addons/Logistics/Nodes/CUP.hpp:336]: please add `canLoadWeapon = 0;`
Whilst the tarp can be hidden, the vehicle usually is closed.
Statics also are glitching into the benches.

**Comment by Bob-Murphy** [comment]
Also please move the nodes a bit so it's not clitching into the front seat
image

Thank you

**Review by Bob-Murphy — APPROVED** [review]
lgtm

**Pull request merged**